### PR TITLE
PRTL-1925. Styling for clinical section values

### PR DESCRIPTION
**Pull request by anncatton**
Displayed values for clinical sections will take whole horizontal space remaining after side tab.

### CODE REVIEW

File: src/packages/@ncigdc/uikit/SideTabs.js [hunk 1]
```diff
@@ -5,8 +5,15 @@ import { withState } from 'recompose';
...
 import Tabs from './Tabs';
 
-const SideTabs = ({ setTab, activeTab, tabs, tabContent, ...props }) => (
-  <Row flex="1">
+const SideTabs = ({
+  setTab,
+  activeTab,
+  tabs,
+  tabContent,
+  containerStyle,
```
Comment (alex-wilmer) [inline src/packages/@ncigdc/uikit/SideTabs.js]: I would add a default here `containerStyle = {}`

Comment (anncatton) [reply src/packages/@ncigdc/uikit/SideTabs.js]: Like:
SideTabs.defaultProps = {
  containerStyle: {},
);

?

Comment (alex-wilmer) [reply src/packages/@ncigdc/uikit/SideTabs.js]: no just default argument

**Comment by anncatton** [comment]
Oh yeah, duh.

**Comment by alex-wilmer** [comment]
also, for future reference use absolute imports like `@ncigdc/utils/whatever`

File: src/packages/@ncigdc/modern_components/ClinicalCard/ClinicalCard.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ import DownloadButton from '@ncigdc/components/DownloadButton';
 import { visualizingButton } from '@ncigdc/theme/mixins';
 import EntityPageVerticalTable from '@ncigdc/components/EntityPageVerticalTable';
 import ageDisplay from '@ncigdc/utils/ageDisplay';
-import { truncate } from '../../utils/string';
+import _ from 'lodash/string';
```
Comment (alex-wilmer) [inline src/packages/@ncigdc/modern_components/ClinicalCard/ClinicalCard.js]: I think just `import { truncate } from 'lodash'` should work?

File: src/packages/@ncigdc/modern_components/ClinicalCard/ClinicalCard.js [hunk 3]
```diff
@@ -117,7 +117,10 @@ export default compose(
                     diagnoses.length > 1
                       ? diagnoses.map(x => (
                           <p key={x.node.diagnosis_id}>
-                            {truncate(x.node.diagnosis_id, 8)}
+                            {_.truncate(x.node.diagnosis_id, {
+                              length: 9,
+                              omission: '…',
```
Comment (alex-wilmer) [inline src/packages/@ncigdc/modern_components/ClinicalCard/ClinicalCard.js]: you can drop this key. the default is `...`

Comment (anncatton) [reply src/packages/@ncigdc/modern_components/ClinicalCard/ClinicalCard.js]: Ok cool. I was wondering if I should bother being explicit.

**Pull request merged**